eventbus.go

```diff
@@ -50,9 +60,19 @@ func (e EventBus) Publish(eventName string, data Data) {
 		return
 	}
 
-	for _, subscriber := range subscribers {
-		subscriber <- data
+	go func(data Data, subscribers []DataChannel) {
+		for _, ch := range subscribers {
+			ch <- data
+		}
+	}(data, subscribers)
```

Comment:
I wonder whether we can do 
```
go func {
		for _, ch := range subscribers {
			ch <- data
		}
	}
```

Reply:
Yes, indeed! I was debugging at the time and forgot to change, I will change back. We desperately need testing for this repository.

Reply:
@arberiii So sorry for my tech debts.